src/api/data_processing.py

```diff
@@ -0,0 +1,172 @@
+import sys, os, logging, datetime
+import pandas as pd
+
+sys.path.append("..")
+from epreader import *
+from datetimeep import *
+
+
+class DataProcessing:
+    def __init__(
+        self,
+        data_path: str = None,
+        data_source: str = None,
+        timestamp_column_name: str = None,
+    ):
+        """Instantiate a data processing object to load datasets and manipulate data before feeding it to the verification process.
+
+        Args:
+            data (str): Path to the data (CSV format) to be loaded for processing.
+            data_source (str): Data source name. Use `EnergyPlus` or `Other`.
+            timestamp_column_name (str): Name of the column header that contains the time series timestamps.
+        """
+        if data_path == None:
+            logging.error("A `data_path` argument should be provided.")
+            if data_source == None:
+                logging.error("A `data_source` argument should be provided.")
+            return None
```

Comment:
should this be separate checks (not nested) to account for the case of data_path is valid and data_source is None?

Reply:
That's correct. Fixed! Thanks!